test_fixtures/task_runner/passing_tasks/test/simple_test.dart

```diff
@@ -1,7 +1,10 @@
+import 'dart:async';
+
 import 'package:test/test.dart';
 
 void main() {
-  test('passes', () {
+  test('passes', () async {
+    await new Future.delayed(const Duration(seconds: 2));
```

Comment:
could you add a comment as to why this is here? I am assuming it's so that the task runner kills the test task since it will complete after the format task fails